build_tools/cmake/iree_lit_test.cmake

```diff
@@ -62,12 +62,7 @@ function(iree_lit_test)
   set(_DATA_DEP_PATHS)
   foreach(_DATA_DEP ${_RULE_DATA})
     string(REPLACE "::" "_" _DATA_DEP_NAME ${_DATA_DEP})
```

Comment:
We still can't use https://github.com/google/iree/blob/b67af5abfb8d3676ba20bba4dbb012d3763742cd/build_tools/cmake/iree_macros.cmake#L191-L197 

here, right? IIRC, there's something about https://github.com/google/iree/blob/b67af5abfb8d3676ba20bba4dbb012d3763742cd/build_tools/cmake/iree_installed_test.cmake#L40-L45

that isn't a target we can add dependencies for?

Reply:
Tests aren't targets in CMake :-/

> Cannot add target-level dependencies to non-existent target
  "iree/samples/custom_modules/dialect/test/custom_ops.mlir.test".